packages/components/cypress/e2e/popover.cy.ts

```diff
@@ -0,0 +1,61 @@
+describe('popover', () => {
+  describe('default', () => {
+    beforeEach(() => {
+      cy.getComponent('popover');
+      // Aria-expanded is set by the web component, therefor it's a good measure to indicate the component is ready
```

Comment:
```suggestion
      // Aria-expanded is set by the web component, therefore it's a good measure to indicate the component is ready
```
Nitpick: Therefor exists but I think you mean more therefore